server.js

```diff
@@ -15,3 +15,4 @@ app.get('/*', function (req, res) {
 app.listen(port);
 
 //    set PORT=3006 && react-scripts start
+// node server.js
```

Comment:
Zazwyczaj nie trzyma się zakomentowanego kodu, chyba że coś ma objaśniać. Wtedy korzysta się z formatu:
// INFO(nazwausera): Tu komentarz informacyjny
// FIXME(nazwausera): Tu komentarz co trzeba poprawić. Np. wiemy że w kolejnej wersji będzie inne API i trzeba to zmienić
// TODO(nazwausera): Tu komentarz typu, Fajnie by było zrefactorować to